src/types.rs

```diff
@@ -537,7 +543,7 @@ impl Date {
 }
 
 /// Used to resolve month abbreviations to their respective values.
-pub fn get_month_for_abbr(month: &str) -> Option<(String, u8)> {
+pub(crate) fn get_month_for_abbr(month: &str) -> Option<(String, u8)> {
```

Comment:
Never seen that before

Reply:
There's also `pub(in <module_name_here>)`, but I've never had a need for that.